opentech/apply/funds/templates/funds/includes/field.html

```diff
@@ -0,0 +1,29 @@
+{% load util_tags %}
+
+{% with widget_type=field|widget_type field_type=field|field_type %}
+
+<div class="form__group">
+    <label for="{{ field.id_for_label }}" class="form__question form__question--{{ field_type }} {{ widget_type }}" {% if field.field.required %}required{% endif %}>
+        <span>{{ field.label }}</span>
+        {% if field.field.required %}
+            <span class="form__required">*</span>
+        {% endif %}
+    </label>
+    {% if field.help_text %}
+        <p class="form__help">{{ field.help_text }}</p>
+    {% endif %}
+    <div class="form__item">
+        {% if widget_type == 'date_input' or widget_type == 'date_time_input' %}
+            <div class="{{ widget_type }}">
+        {% endif %}
+
+        {{ field }}
+
+        {% if field.errors %}<h6 class="form__error-text">{{ field.errors.as_text }}</h6>{% endif %}
+        <label for="{{ field.id_for_label }}"></label>
```

Comment:
I'm not sure about this, but copied from https://github.com/OpenTechFund/opentech.fund/pull/63/files#diff-82db770bd499eb2f6347c368c70dc55cL41